### ISSUE TITLE
Install biohazard-tools via conda instead of providing binaries

### ISSUE THREAD

**Issue by alexhbnr** [alexhbnr/mitoBench-ancientMT#4]

**Comment by wjv** [comment]
Did you ever manage this, @alexhbnr? I'm having problems trying to do same.

**Comment by alexhbnr** [comment]
No, I haven't tackle this. But the code here needs anyway an update at some point soon to make the pipeline more universal and then I will give it a try. But it seems that this won't be easy at all.
